packages/common/src/authentication/AuthContext.tsx

```diff
@@ -218,8 +221,12 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
     const timer = window.setInterval(() => {
       const authCookie = getAuthCookie();
       const { token } = authCookie;
+      const isInitScreen = matchPath(
```

Comment:
added this in because the only way I could see of determining that the sync settings had been applied and the server was back up and running was to poll for the `Unauthenticated` response to the settings query. 

Without this little addition, the cookie is checked, the auth error set and the polling returns too early 🙄